packages/eslint-config-vivy/index.js

```diff
@@ -16,6 +16,16 @@ module.exports = {
       { extensions: [".js", ".jsx", ".tsx"] },
     ],
     "prettier/prettier": ["error"],
+    "react/jsx-props-no-spreading": 1,
+    "import/extensions": 0,
```

Comment:
Whats the reason for disabling this after adding it?

Reply:
this is needed because it conflicts with a TS rule NOT to use extensions!

Reply:
Ah right! I thought this said `import/resolver`